src/Core/Migration/V6_5/Migration1687462843ProductManufacturerMediaThumbnails.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_5;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+/**
+ * @internal
+ */
+#[Package('core')]
+class Migration1687462843ProductManufacturerMediaThumbnails extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1687462843;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $thumbnailSizes = $this->upsertThumbnailSizes($connection);
+
+        $configurationId = $connection->fetchOne(
+            'SELECT media_folder_configuration_id FROM media_folder WHERE name = :name',
+            ['name' => 'Product Manufacturer Media']
+        );
+
+        if (!$configurationId) {
+            return;
+        }
+
+        foreach ($thumbnailSizes as $thumbnailSizeId) {
+            $connection->executeStatement('
```

Comment:
outside of loop prepare the query ($connection->prepare($sql);` and then execute it inside the loop